modules/openxr/extensions/openxr_opengl_extension.cpp

```diff
@@ -167,8 +167,8 @@ void OpenXROpenGLExtension::get_usable_swapchain_formats(Vector<int64_t> &p_usab
 	p_usable_swap_chains.push_back(GL_SRGB8_ALPHA8);
 	p_usable_swap_chains.push_back(GL_RGBA8);
 #else
-	p_usable_swap_chains.push_back(GL_SRGB8_ALPHA8_EXT);
-	p_usable_swap_chains.push_back(GL_RGBA8_EXT);
+	p_usable_swap_chains.push_back(GL_SRGB8_ALPHA8);
+	p_usable_swap_chains.push_back(GL_RGBA8);
```

Comment:
The three `#if`/`#elif`/`#else` seem pretty redundant now since it's all the same code.